charts/consul/templates/server-statefulset.yaml

```diff
@@ -192,6 +192,13 @@ spec:
             - |
               CONSUL_FULLNAME="{{template "consul.fullname" . }}"
 
+              {{- if (and .Values.dns.enabled .Values.dns.enableRedirection)}}
+              export kube_dns_service_ip=$(cat /etc/resolv.conf | grep nameserver | cut -d' ' -f2)
+              if [[ -z $kube_dns_service_ip ]]; then
+                  echo "Unable to get kube-dns Service IP"
+              fi
```

Comment:
🤔 It's a bit strange that we just echo here and don't do anything. 

I'm also thinking that it'll probably be ok to assume the nameserver(s) will always be there on k8s.